sktree/tree/_honest_tree.py

```diff
@@ -431,29 +425,31 @@ def _inherit_estimator_attributes(self):
         self.n_outputs_ = self.estimator_.n_outputs_
         self.tree_ = self.estimator_.tree_
 
-    def _empty_leaf_correction(self, proba, normalizer):
+    def _empty_leaf_correction(self, proba, pos=0):
```

Comment:
Can you just add a short docstring to describe what's going on? I'm reading these lines and having trouble figuring out what `pos` does actually :/

Reply:
`pos` indicates the class dimension of y, so that posterior corrections only work on that dimension in multi-output cases.